call.sh

```diff
@@ -1,13 +1,26 @@
 #!/bin/bash
-declare -a Array
-curl https://www.cnn.com/ | grep -Eo '*\/[0-9]{4}\/[0-9]{2}\/[0-9]{2}\/[a-z/-]+.html' | awk '$0="https://www.cnn.com"$0' | mapfile -t Array
+depth=1
```

Comment:
Ideally this would be an argument to make it easy to modify the depth i.e.

```
sh call.sh https://www.cnn.com 5
sh call.sh https://www.cnn.com 10
```